static/js/components/Dashboard.js

```diff
@@ -2,9 +2,14 @@
 import React from 'react';
 import CourseList from './CourseList';
 import UserImage from './UserImage';
+import _ from 'lodash';
 
 class Dashboard extends React.Component {
 
+  formatId(id) {
+    return id ? `MMM${_.padStart(id.toString(), 6, '0')}` : undefined;
```

Comment:
I can go both ways here. It seems like we should only be providing raw data from the server (at least through REST APIs) and do all formatting on the frontend. On the other hand you could argue that the `MMM...` string is our identifier and therefore a piece of raw data.

Whichever you choose, we should have a test for this